.github/workflows/tests.yaml

```diff
@@ -0,0 +1,51 @@
+name: tests
+
+on:
+  push:
+    branches: [master]
```

Comment:
probably better to run these on every push, so they run on tags and branches too